roles/reproducer/tasks/main.yml

```diff
@@ -142,6 +142,33 @@
       }}
   failed_when: false
 
+- name: Ensure custom networks can be routed outside to the hypervisor
+  when:
+    - cifmw_reproducer_nat_public_interfaces | default ('false') | bool
+  become: true
+  block:
+    - name: Enable masquered over public interfaces
```

Comment:
FYI, this part can be dopped. It is managed elsewhere. See: https://github.com/openstack-k8s-operators/ci-framework/pull/1928